google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     compile ('com.google.cloud.tools:appengine-plugins-core:' + toolsLibVersion) {
         exclude group: 'com.google.guava', module: 'guava'
     }
+    compile 'org.yaml:snakeyaml:1.18'
```

Comment:
is it correct to have this on the ultimate module's build file? what from ultimate needs this? I thought that since this is all flex related, and flex can be run on community, that you could add this to the "regular" build file only.